static/src/javascripts-legacy/projects/commercial/modules/article-body-adverts.js

```diff
@@ -174,7 +174,9 @@ define([
     }
 
     function addSlots(totalCount) {
-        qwery('.ad-slot--inline').forEach(addSlot);
+        qwery('.ad-slot--inline').forEach(function (slot) {
```

Comment:
Reminder: the signature of an array iteration callback is `function(element, index, originalArray)`